30daysofscala/src/test/scala/example/HelloSpec.scala

```diff
@@ -0,0 +1,10 @@
+package example
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+class HelloSpec extends AnyFlatSpec with Matchers {
+  "The Hello object" should "say hello" in {
```

Comment:
Update the test case to also check that hello.greeting returns "Welcome to 30 days of Scala"

Reply:
ok